connector/datadogconnector/go.mod

```diff
@@ -0,0 +1,88 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/connector/datadogconnector
+
+go 1.20
+
+require (
+	github.com/DataDog/datadog-agent/pkg/proto v0.48.0-beta.1
```

Comment:
@songy23 can you check what version should we use here and if we need replace statements?

Reply:
We should use `v0.48.0-beta.1` , if the new component is included in dependabot config then we need replace as well.